### DISCUSSION TITLE
Private key (longer than 255 chars) needed in environmental variables

### DISCUSSION THREAD

**Discussion by toychicken** [beginner-corp/begin-community#62] [Q&A] [answered]
Hi - I'm using Begin to send data back to an existing DB, via a Google Cloud service account. However, to authenticate, I need to send the private key (amongst other things). Normally I'd put this in an environmental variable, but it's 1077 chars, and the Begin.com env variables are limited to 255 characters. Obviously, I'd rather not put this in the repo, so does anyone have any suggestions / tips for how to handle this kind of scenario. 

**NB** For clarity, I can't change the authentication method (not mine), so it's really down to the best way to handle bigger env variable payloads (or some other approach).

**Comment by ryanblock** [comment, marked as answer]
You should be good to go! Environment variables **names** are limited to 255 chars, while values are limited to 4KB each. (We did have some lingering copy that said 1KB, that'll be updated in a few minutes.)

**Reply by toychicken** [reply]
Thanks! I did wonder if I'd misunderstood 😊
